conjur/controller/login_controller.py

```diff
@@ -96,3 +97,12 @@ def get_api_key(self, conjurrc: ConjurrcData):
                                                                      self.credential_data,
                                                                      self.user_password,
                                                                      conjurrc)
+
+    def _get_password_prompt(self, conjurrc: ConjurrcData = None):
```

Comment:
Very small nit, and maybe serves a purpose that I'm not aware of (in which case feel free to ignore) - it seems slightly redundant to have both `get_password` and `_get_password_prompt` providing a default value of `None` to conjurrc when the value is not changed between calls

Reply:
Yeah good point. I fixed that.